.github/workflows/main.yml

```diff
@@ -16,8 +16,6 @@ jobs:
       with:
         fetch-depth: 0
     - uses: pnpm/action-setup@v4
-      with:
-        version: 8
```

Comment:
we are still using pnpm v8 for everything. This change will break nearly all our commits.

@nklomp not time to migrate to v9? It's just updating the CI ymls, then everyone can upgrade pnpm.

Reply:
oh wait, I do not think this is an issue for this project as this is not publishing with lerna. Other projects won't publish when you use a different pnpm version because you will end up with an modified / uncommited version of pnpm-lock.yaml

Reply:
It's still using v8. But i added it to the package.json so i can easily install the correct version per project (using corepack). But then ci started to complain that it was already defined in package.json (it reads from there)